### PR TITLE
Fix missing groups

### LINKED ISSUE #495: Users who are part of "review" and "signoff" groups do not get shown the "request review" or "grant review" buttons
I think #471 might have broken something with the conditions under which the "request review" or "grant review" buttons are even shown. The principals for the editors/reviewers are correctly shown in the homepage, and the collection metadata can be edited, but the buttons are mysteriously absent.

### PR DESCRIPTION
**Pull request by glasserc**
This should fix #495.

### CODE REVIEW

**Comment by glasserc** [comment]
I wanted to rename `editors_group` and `reviewers_group` to camelcase, but doing so requires changes at both this level and the capabilities level, which seems more invasive than I am comfortable with.

**Comment by glasserc** [comment]
This might also introduce some bugs, and needs tests.

**Comment by leplatrem** [comment]
screenshot from 2018-03-27 18-02-58

File: test/plugins/signoff/sagas_test.js [hunk 1]
```diff
@@ -86,6 +89,82 @@ describe("Signoff plugin sagas", () => {
...
+                bid: "stage",
+                cid: "source-plugins",
+                changes: {},
+                lastAuthor: "Last Author",
+                lastEditor: "Last Editor",
+                lastEditorComment: "Last Editor Comment",
+                lastReviewer: "Last Reviewer",
+                lastReviewerComment: "Last Reviewer Comment",
+                editors_group: "{collection_id}_editors_who_wear_cool_sneakers",
+                reviewers_group: undefined,
```
Comment (leplatrem) [inline test/plugins/signoff/sagas_test.js]: Do we want this attribute as `undefined` or should the attribute be omitted?

**Comment by leplatrem** [comment]
@glasserc You can merge this :)

**Pull request merged**